src/main/kotlin/io/stud/forest/book_a_cinema_ticket_app/domain/RoomOccupancy.kt

```diff
@@ -0,0 +1,9 @@
+package io.stud.forest.book_a_cinema_ticket_app.domain
+
+class RoomOccupancy(
+        val seats: List<SeatOccupancy>
+) {
+    val rows: Long = seats.map { it.seat.row }.maxOrNull() ?: 0 // consider Map
```

Comment:
A zwykły `max()` by tu nie starczył? Jaki jest sens jeśli mamy 0?

Reply:
`max` wskazuje mi jako deprecated. 
```
@Deprecated("Use maxOrNull instead.", ReplaceWith("this.maxOrNull()"))
@DeprecatedSinceKotlin(warningSince = "1.4", errorSince = "1.5")
```

>  Jaki jest sens jeśli mamy 0?
Można by rzucić wyjątkiem, w tej sytuacji. Ale przeniósł bym to wymaganie do:
```
init{
    required()
}
```

>